### PR TITLE
added python-pysynphot for x86_64

### PR DESCRIPTION
**Pull request by migueldvb**
added python-pysynphot for x86_64

### CODE REVIEW

File: python-pysynphot/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Maintainer: Astro Benzene <universebenzene at sina dot com>
+pkgbase=python-pysynphot
+_pyname=${pkgbase#python-}
+pkgname=("python-${_pyname}" "python2-${_pyname}" "python-${_pyname}-doc")
```
Comment (petronny) [inline python-pysynphot/PKGBUILD:4]: In this case, you need to add a file called `package.list`
and list all the `pkgname` in it, eg. https://github.com/arch4edu/arch4edu/blob/master/python-av/package.list

Comment (migueldvb) [reply python-pysynphot/PKGBUILD:4]: Ah, I see. Added the `package.list` file for this PKGBUILD.

**Comment by petronny** [comment]
Hi, the build failed last night.
I assume you have recieved the email sent by calil.

Sorry for the Chinese in the email.
We will do the i18n in the future.

The main error in the build log is:
```
==> Installing missing dependencies...
error: target not found: python-astropy
```

In this case, you should added 
```
repo_depends = ['python-astropy']
```
in `lilac.py`.
Also since this is a python package, you should add
```
{'alias': 'python'}
```
in `update_on` to automatically rebuild the package when python has an major update.
[Here](https://github.com/arch4edu/arch4edu/blob/master/python-librosa/lilac.py) is an example.

I'll fix it this time.

**Comment by migueldvb** [comment]
Yes, I have received the email with the build log. Thank you for taking
care of the problem!

**Comment by petronny** [comment]
35ab6f54eadd3974148c9fd37cd0f9b

**Pull request merged**